esmvaltool/utils/color-tables/show_color_tables.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Please create a function and optionally add a
```python
if __name__ == '__main__':
    run_some_function()
```
block, so the file can be imported as a module.

Reply:
Done